panel/cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+	video: false,
+
+	e2e: {
+		baseUrl: "http://sandbox.test",
+		specPattern: "src//**/*.e2e.js",
```

Comment:
What does the double-slash in the path do?

Reply:
Good question, tbh cypress migrated its own settings during first run - probably can be just one